src/ast/passes/codegen_llvm.cpp

```diff
@@ -1147,6 +1147,15 @@ void CodegenLLVM::visit(Call &call)
   {
     expr_ = b_.getInt64(call.vargs->at(0)->type.GetSize());
   }
+  else if (call.func == "offsetof")
+  {
+    auto identifier = dynamic_cast<Identifier *>(call.vargs->at(1));
+    if (identifier)
```

Comment:
Perhaps instead of:

```
if (identifier)
{
  ...
}
```

Do a:

```
assert(identifier);
...
```

At this point in codegen it's an internal bug if the argument is not an identifier

Reply:
Fixed, thank you!!